core/modules/dblog/css/dblog.css

```diff
@@ -68,18 +68,22 @@
   width: 30px;
   height: 30px;
   border-radius: 15px;
+  background: #ccc url(../../../misc/message-32-info.png) no-repeat center center;
+  background-size: 16px;
 }
+.admin-dblog .dblog-info .icon:before {
+  background: #004570 url(../../../misc/message-32-info.png) no-repeat center center;
+  background-size: 16px;
+}
+.admin-dblog .dblog-debug .icon:before,
 .admin-dblog .dblog-warning .icon:before {
-  background: #FCE400 url(../../../misc/message-16-warning.png) no-repeat center center;
+  background: #FCE400 url(../../../misc/message-32-warning.png) no-repeat center center;
+  background-size: 16px;
 }
 .admin-dblog .dblog-error .icon:before,
 .admin-dblog .dblog-critical .icon:before,
 .admin-dblog .dblog-alert .icon:before,
 .admin-dblog .dblog-emergency .icon:before {
-  background: #EE3D23 url(../../../misc/message-16-error.png) no-repeat center center;
-}
-.admin-dblog .dblog-debug .icon:before,
-.admin-dblog .dblog-info .icon:before,
-.admin-dblog .dblog-notice .icon:before {
```

Comment:
we loose notice ?

Reply:
Looks like it...

Reply:
No, the default is notice now, instead. See line 71.

Reply:
> we loose notice ?

Nope, see line 71.